components/private_channel/browser/private_channel_service.cc

```diff
@@ -0,0 +1,320 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+
+#include "brave/components/private_channel/browser/private_channel_service.h"
```

Comment:
header for this file should go at the top, then a space and any system headers and then all remaining headers in alphabetical order

Reply:
- [x] fixe order imports


Running `clang_format` helped here